Added software detail pages

ikke09 opened the pull request:
Added extra subpages for every software product. For now  I just did this for the first four products in the list. If this first request is getting accepted I will also adding pages for the other products.

The Link of the product opens now these subpages. The page gives a quick overview over the product, so that users can get a first impression of it. Also additional links like help or the donation sides. 

Unfortunately i was not able to test the new pages on my local windows mashine. So hopefully the pages are all working correct.

dschuermann commented on _software/01_02-gpg4win.md:
```diff
@@ -0,0 +1,25 @@
+* Developer/Publisher: Community
+* Application field: Wide offer on encryption tools 
+* License: Open Source (GNU GPL)
+* Price: All tools are freeware.[Donations](https://www.gpg4win.de/donate.html) are desired.
```
"freeware" is a charged term. better use "free of charge"

dschuermann commented on _software/01_02-gpg4win.md:
```diff
@@ -0,0 +1,25 @@
+* License: Open Source (GNU GPL)
+* Price: All tools are freeware.[Donations](https://www.gpg4win.de/donate.html) are desired.
+* Web: [https://www.gpg4win.de/index.html](https://www.gpg4win.de/index.html
+* Help: Help is given through the communitys
```
please use a spell checker: it's "communities"

dschuermann commented on _software/01_03-p≡p.md:
```diff
@@ -0,0 +1,27 @@
+
+# p≡p
+
+p≡p is a crossplattform implementation of the OpenPGP Standard. It is available for Android although for Microsoft Outlook. The focus is on a secure connection and an automatic encryption.
```
wrong use of "although"

dschuermann replied:
and better use a filename without special characters

dschuermann commented on _software/01_03-p≡p.md:
```diff
@@ -0,0 +1,27 @@
+* License: Open Source (GNU GPL)
+* Price: 
+	* Android app: Free (Beta)
+	* Outlook Add-In: about $20
```
better be precise here

dschuermann commented on _software/01_04-enigmail.md:
```diff
@@ -0,0 +1,25 @@
+{% include toc %}
+
+# Enigmail
+Enigmail is an OpenPGP addon for Mozilla Thunderbird. This product brings the encryption features provieded by GnuPG into Thunderbird and allows to sign and encrypt all your emails.
```
spell-check! "provided"

dschuermann commented:
Looks okay to me now. I will merge if this is available for all applications on the email page.

dschuermann commented:
@ikke09 Can you provide this structure for all email apps? Then we can do this change!

ikke09 commented:
Sorry for the delay. 

Added the details pages for every email application.
I only skipped the email provider because they have no own software products.

Also removed the include statements on every page.

dschuermann commented:
@ikke09 Merged and fixed in https://github.com/OpenPGP/openpgp.github.io/commit/4558be82b707ad437d769d1905e254512a3287a7
